scrapers/errata.py

```diff
@@ -207,3 +237,57 @@ def get_attached_bugs(self, advisory_id):
         """.format(advisory_id)
         log.info('Getting Bugzilla bugs tied to the advisory with ID {0}'.format(advisory_id))
         return self.teiid.query(sql)
+
+    def get_koji_build(self, build_id):
+        """
+        Query Teiid to find the Koji builds attached to a specific advisory.
+
+        :param int build_id: the build ID
+        :return: a list of a dictionaries
+        :rtype: list
+        """
+        sql = """\
+            SELECT
```

Comment:
Only select the columns you need for `is_container_build` to run. This will improve performance and make the code easier to read.